tests/test_inspector.py

```diff
@@ -581,7 +643,20 @@ def test_inspect_nwbfile_dandi_config(self):
         self.assertCountEqual(first=test_results, second=true_results)
 
 
-class TestDANDIConfig(TestInspector):
+class TestInspectorAPIAndCLIZarr(TestInspectorAPIAndCLIHDF5):
+    BackendIOClass = BACKEND_IO_CLASSES["zarr"]
+    true_report_file_path = EXPECTED_REPORTS_FOLDER_PATH / "true_nwbinspector_default_report_zarr.txt"
+    skip_validate = True
+
+    # TODO: undo this override
+    def test_iterable_check_function(self):
+        pass
```

Comment:
This one is mysterious

It only fails when you run the entire testing suite, including in CI: https://github.com/NeurodataWithoutBorders/nwbinspector/actions/runs/11040199954/job/30667657087?pr=513

But run in debug mode, console, or in isolation it's fine

So actual test in truth, passes, but some unittest mystery throws it off in CI, so just skipping for now

@stephprince could you raise an issue and figure it out when you get the time?

Reply:
could it be that `iterable_check_function` is getting registered twice? I was able to reproduce the failure locally by running pytest from the command line, and I believe moving the `iterable_check_function` definition and registration outside of the class solves the issue.

Reply:
Right you are, thanks for figuring that out!